src/git/models/repository.ts

```diff
@@ -647,12 +647,12 @@ export class Repository implements Disposable {
 
 	@gate()
 	@log<Repository['getMainRepository']>({ exit: r => `returned ${r?.path}` })
-	async getMainRepository(): Promise<Repository | undefined> {
+	async getMainRepository(options?: { detectNested?: boolean; force?: boolean }): Promise<Repository | undefined> {
```

Comment:
Can we avoid taking the options in here, and just pass `{ detectNested: false, force: true }` to the `getOrOpenRepository` call -- since we always want `getMainRepository` to apply those

Reply:
Done in commit: [Always sets force and no-nested from getMainRepository](https://github.com/gitkraken/vscode-gitlens/pull/2544/commits/1de358c86fae0a5c23f57ea4b0ecdd55ffd48805)